action.yaml

```diff
@@ -0,0 +1,33 @@
+name: 'Local Docker image scan'
+description: 'Scan Docker images using Trivy locally'
+inputs: 
+  image-name:
+    description: 'Docker image to scan'
+    required: true
+  whitelist-file:
+    description: 'Path to the whitelist file'
+    required: false
+  whitelist-file-branch:
+    description: 'Branch from which whitelist file is to be downloaded'
+    default: 'master'
+    required: false
+  username:
+    description: 'Username to authenticate to the Docker registry'
+    required: false
+  password:
+    description: 'Password to authenticate to the Docker registry'
+    required: false
+  github-token:
+    description: 'Github token to download whitelist file'
+    default: ${{ github.token }}
+  add-CIS-checks:
+    description: 'Add CIS checks to build secure and best pracitce Docker images'
+    default: 'true'
+
```

Comment:
Add an input severity-threshold. Users can specify this as [UNKNOWN, LOW, MEDIUM, HIGH, CRITICAL]. Based on this we set TRIVY_SEVERITY env variable.
eg. User selects MEDIUM, we set TRIVY_SEVERITY to MEDIUM, HIGH, CRITICAL

Reply:
Added